docs/RFCS/range_leases.md

```diff
@@ -101,6 +114,23 @@ node ID and epoch match the last committed lease, the command will be
 applied; otherwise it will be rejected.
 
 
+# Performance implications
+
+We expect traffic proportional to the number of nodes in the system.
+With 1,000 nodes and a 9s liveness duration threshold, we expect every
+node to do a conditional put to update the heartbeat timestamp every
+7.2s. That would correspond to ~140 reqs/second, a not-unreasonable
+load for this function.
+
+We still require the traditional expiration-based range leases for any
+ranges located at or before the liveness table's range. This might be
+problematic in the case of meta2 address record ranges, which are
+expected to proliferate in a large cluster. This lease traffic could
```

Comment:
Well you end up with some historical versions and other dross, so I agree somewhere between 10 and 50 meta2 ranges. Doesn't seem worth the effort. I added a note.